weio/tests/test_bladed.py

```diff
@@ -0,0 +1,25 @@
+import os
+import numpy as np
+import re
+import pandas as pd
+import weio
+import unittest
+from helpers_for_test import MyDir, reading_test 
+from weio.bladed_out_file import Read_bladed_file
+
+
+
+class Test(unittest.TestCase):
+ 
+    def test_001_read_all(self):
+        reading_test('bladed*.*', Read_bladed_file)   
```

Comment:
You can comment this line and replace with  "pass" for now. 
This basically try to read all the files in the folder that starts with bladed. In this case it won't work since the files are in subfolders. Also bladed has a lot of different files so that won't work, unless we do a small trick to actually read the PJ file even if we are given a $ or % file.